src/app/features/management/group-management-view/group-management-view.component.ts

```diff
@@ -1,9 +1,97 @@
+import { ActivatedRoute, Router } from '@angular/router';
 import { Component } from '@angular/core';
+import { orderDirFromSort, Paginated, PaginationParams } from '../../../core/util/store';
+import { EMPTY, tap } from 'rxjs';
+import { Loader } from '../../../core/util/loader';
+import { GroupService, GroupSort } from '../../../core/services/group.service';
+import { Group } from '../../../core/model/group';
+import { MDSError, MDSErrorCode } from '../../../core/util/errors';
+import { Sort } from '@angular/material/sort';
+import { Operation } from '../../../core/model/operation';
+import { OperationService } from '../../../core/services/operation.service';
 
 @Component({
   selector: 'app-group-management-view',
   templateUrl: './group-management-view.component.html',
   styleUrls: ['./group-management-view.component.scss'],
 })
 export class GroupManagementView {
+  columnsToDisplay = ['title', 'description', 'operation'];
+  pagination?: PaginationParams<GroupSort>;
+  loadedGroups?: Paginated<Group>;
+
+  constructor(private router: Router, private route: ActivatedRoute, private groupService: GroupService, private operationService: OperationService) {
+  }
+
+  retrieving = new Loader();
+
+  refresh(): void {
+    this.retrieving.loadFrom(() => {
+      if (!this.pagination) {
+        return EMPTY;
+      }
+      return this.groupService.getGroups(this.pagination, {});
+    }).subscribe(result => (this.loadedGroups = result));
+  }
+
+  navigateToGroup(groupId: string): void {
+    this.router.navigate([groupId], { relativeTo: this.route }).then();
+  }
+
+  createGroup() {
+    this.router.navigate(['create'], { relativeTo: this.route }).then();
+  }
+
+  /**
+   * Called when pagination is updated.
+   * @param pagination
+   */
+  page(pagination: PaginationParams<string>): void {
+    this.pagination = pagination.mapOrderBy(GroupManagementView.mapOrderBy);
+    this.refresh();
+  }
+
+  asGroup(g: Group): Group {
+    return g;
+  }
+
+  asOperation(o: Operation): Operation {
+    return o;
+  }
+
+  getOpTitle(g: Group): string {
+    if (!g.operation) {
+      return '';
+    }
+    let retVal = '';
+    this.operationService.getOperationById(g.operation).pipe(tap(operation => {
+      retVal = operation.title;
+    })).subscribe();
+    return retVal;
+  }
+
```

Comment:
Not proceeding with review.
- `getOpTitle`: why the abbreviation?
- `retVal`: Why the abbreviation? Should describe its content!
- method is simply broken. it's a function with async operations and always returns `''`. retrieve when operations where retrieved, save in map or better: create custom entities for rows that are shown in the table with nullable operation title. This will be set when operation is retrieved.
- never ever ever perform network requests from template! i don't know if you tested this but my machine is going up in flames. hint: open the network tab.
- and even if you didn't: why return an empty string? it's not even documented. simply take the operation id directly -> `getOperationTitleById` but as already mentioned, this makes no sense.